.travis.yml

```diff
@@ -6,28 +6,34 @@ cache:
         - extra/*/vendor
         - $HOME/.composer/cache/files
 
+env:
+    global:
+        - SYMFONY_PHPUNIT_REMOVE_RETURN_TYPEHINT=1
+        - SYMFONY_PHPUNIT_DISABLE_RESULT_CACHE=1
```

Comment:
Except for this line which can be kept, this commit should be basic reverted when merging in 3.x.